resources/puppetlabs/lein-ezbake/template/global/ext/cli/reload.erb

```diff
@@ -2,32 +2,34 @@
 set +e
 
 restartfile=<%= EZBake::Config[:restart_file] %>
-timeout=<%= EZBake::Config[:start_timeout] %>
+reload_timeout=${RELOAD_TIMEOUT:-<%= EZBake::Config[:reload_timeout] %>}
+timeout=$reload_timeout
```

Comment:
This does an `-eq` expression below so if the `timeout` were ever anything that doesn't coerce to an integer expression, the `-eq` check would blow up anyway.  Given that, I was a bit apathetic about quoting the timeout.  Wouldn't seem to cause any harm, though, so I can do that.